web_src/less/index.less

```diff
@@ -17,6 +17,7 @@
 @import "./chroma/light.less";
 @import "./codemirror/base.less";
 @import "./codemirror/light.less";
+@import "./console/console.less";
```

Comment:
Yes, this will include it in the `index.css` chunk. We could dynamically import it via JS when needed, but I think the CSS is small enough to not go through the pain of detecting when it's needed so I'd say it's fine to keep it in the index chunk.